package.json

```diff
@@ -8,80 +8,80 @@
   },
   "browser": "app/init.jsx",
   "dependencies": {
-    "@babel/polyfill": "^7.4.4",
```

Comment:
@fayceltouili, you may want to discard all of your changes to both package.json and package-lock.json, since you've bumped the versions of many libraries, and that might be pretty dangerous, since there could be incompatibilities that we haven't tested for.

If you need to bump the versions of a few specific dependencies, that should be fine, but I wouldn't recommend bumping the versions of nearly every library here.

Reply:
@richardxia, you've right I will add package.json and package-lock.json to .gitignore

Reply:
Looks like these changes to package.json and package-lock.json are still present in this branch.

BTW, you shouldn't add either of these files to .gitignore; they are important to make sure we are all synchronized in terms of which versions of packages we are using.

You should be able to restore these back to what they were before with the following sequence of commands:

```sh
# Ensure latest commits from GitHub are fetched
git fetch

# Change package.json and package-lock.json back to what they were on the master branch
git restore --source=origin/master package.json package-lock.json

# Add and commit the changes
git add package.json package-lock.json
git commit -m "Restore original package.json and package-lock.json"
```

Reply:
Hi @richardxia I've fixed package.json and package-lock.json changes. Thank you!